sci-libs/libccd/libccd-9999.ebuild

```diff
@@ -19,20 +19,16 @@ HOMEPAGE="http://libccd.danfis.cz/
 
 LICENSE="BSD"
 SLOT="0"
-IUSE="+double-precision doc +shared test"
+IUSE="+double-precision doc test"
 RESTRICT="!test? ( test )"
 
-RDEPEND=""
-
-DEPEND="${RDEPEND}
-	doc? ( dev-python/sphinx )
-"
+DEPEND="doc? ( dev-python/sphinx )"
 
 src_configure() {
 	local mycmakeargs=(
 		-DBUILD_DOCUMENTATION=$(usex doc ON OFF)
-		-DBUILD_SHARED_LIBS=$(usex shared ON OFF)
 		-DENABLE_DOUBLE_PRECISION=$(usex double-precision ON OFF)
+		-DBUILD_SHARED_LIBS=ON
 	)
 
 	local CMAKE_BUILD_TYPE="Release"
```

Comment:
I don't know why this was added here, it seems to build fine for me with Gentoo build type, should I just drop this?

Reply:
yes drop

Reply:
Done